File: Week_01/Twosum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 0 ; i < nums.length ; i++) {
            for ( int j = i+1 ; j < nums.length ; j ++ ) {
                if ( nums[j] == target - nums[i]) {
                    
                    return new int[]{i,j};
                }
            }
        }
        throw new IllegalArgumentException("No two sum solution");
    }
}

public class Twosum{
    public static void main(String args[]){
        Solution s = new Solution();
        int num [] = {2,7,11,15};
        int t = 9 ;
        s.twoSum(num,t);
     
    }
}

File: Week_01/twosum.py
class Solution:
    def twoSum(self, nums, target):
        i = 0
        j = i+1
        indices = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if nums[i] == target - nums[j] and i !=j:
                   indices.append(i)
                   indices.append(j)
                   return indices
                j += 1
            i += 1
        
        

test  = [2,7,11,15]
test2 = [3,2,4]
t_target = 6
s = Solution()
r = s.twoSum(nums=test2,target = t_target )
print(r)